File: ecommerce/src/main/java/com/ecommerce/ecommerce/model/Product.java
package com.ecommerce.ecommerce.model;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor 
@NoArgsConstructor
public class Product {
    private Long id;
    private String name;
    private String description;
    private double price;
    private String images;
    private String brand;
    private Category category;
    
    private List<Purchase> purchases;
}

File: src/app/models/Product.ts
export class Product{
    Id?: number;
    brand!:string;
    description!:string;
    images!:string;
    name!:string;   
    price!:number;

    constructor(Id : number,brand : string, description:string,
    images:string,name:string,price:number){
            this.Id=Id;
            this.brand=brand;
            this.description=description;
            this.images=images;
            this.name=name;
            this.price=price;
        }

}


